### ISSUE TITLE
Don't see any blocks in 1.20.2, cannot edit anything

### ISSUE THREAD

**Issue by Gud-Git** [Amulet-Team/Amulet-Map-Editor#977]
## Bug Report
Don't see any blocks in 1.20.2, cannot edit anything

### Current Behaviour:
After creating a new world in 1.20.2 only a red checkerboard is visible. It is not possible to edit anything. Or to paste anthing from other working worlds.

### Expected behavior:
Seeing the blocks of the world.

### Steps To Reproduce:
Create a new world in 1.20.2
Exit said world.
Open said world in Amulet.
Look around.

### Environment:

- OS: Windows 
- Minecraft Platform: Java 
- Minecraft Version: 1.20.2
- Amulet Version: V0.10.19

### Screenshots

![image](https://github.com/Amulet-Team/Amulet-Map-Editor/assets/108307919/7e7b836c-1336-44a2-b7c6-32a670b04f8d)

### Worlds
Any world in 1.20.2

**Comment by gentlegiantJGC** [comment]
Building a new release to fix this now.

**Comment by Scroogeee** [comment]
@gentlegiantJGC I can reproduce the issue in Amulet 0.10.20 with Minecraft 1.20.2 on OSX:

<img width="1485" alt="Bildschirmfoto 2023-09-25 um 11 39 07" src="https://github.com/Amulet-Team/Amulet-Map-Editor/assets/30441849/3e1b7aa9-2f31-43bc-845f-076a1fa9a715">

I think the issue should be reopened.

**Comment by gentlegiantJGC** [comment]
@Scroogeee please can you share your console logs

**Comment by gentlegiantJGC** [comment]
@Scroogeee I assume you are running from source. Please can you share `pip list`

**Comment by Scroogeee** [comment]
> @Scroogeee I assume you are running from source. Please can you share `pip list`

```
Package                 Version
----------------------- -------
amulet-core             1.9.17
amulet-leveldb          1.0.0b5
amulet-map-editor       0.10.20
amulet-nbt              2.1.1
minecraft-resource-pack 1.3.5
mutf8                   1.0.6
numpy                   1.25.1
packaging               23.1
Pillow                  10.0.0
pip                     23.2.1
platformdirs            3.9.1
portalocker             2.7.0
PyMCTranslate           1.2.15
PyOpenGL                3.1.7
PyOpenGL-accelerate     3.1.7
setuptools              68.0.0
six                     1.16.0
wheel                   0.38.4
wxPython                4.2.1
```

**Comment by Scroogeee** [comment]
> @Scroogeee please can you share your console logs

Before opening any world:
```
Traceback (most recent call last):
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet_map_editor/api/framework/app.py", line 41, in <lambda>
    lambda evt: update_check.UpdateDialog(
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet_map_editor/api/framework/update_check.py", line 34, in __init__
    super().__init__(parent)
TypeError: Dialog(): arguments did not match any overloaded call:
  overload 1: too many arguments
  overload 2: argument 1 has unexpected type 'AmuletApp'
  ```
  
  After opening the 1.20.2 world I get loads of the same exception. This seems to be one of them:
```
Traceback (most recent call last):
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet/api/wrapper/format_wrapper.py", line 442, in _safe_load
    return meth(*args)
           ^^^^^^^^^^^
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet/api/wrapper/format_wrapper.py", line 484, in _load_chunk
    interface, translator, game_version = self._get_interface_and_translator(
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet/api/wrapper/format_wrapper.py", line 250, in _get_interface_and_translator
    interface = self._get_interface(raw_chunk_data)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet/level/formats/anvil_world/format.py", line 326, in _get_interface
    return Interfaces.get(key)
           ^^^^^^^^^^^^^^^^^^^
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet/level/loader.py", line 62, in get
    object_id = self.identify(identifier)
                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/scrooge/opt/anaconda3/envs/amulet/lib/python3.11/site-packages/amulet/level/loader.py", line 72, in identify
    raise LoaderNoneMatched(
amulet.api.errors.LoaderNoneMatched: Could not find a matching interface for ('java', 3578)
ERROR - Error loading chunk -1 5 minecraft:overworld
```

**Comment by gentlegiantJGC** [comment]
`amulet-core             1.9.17`
The issue was fixed in `1.9.19`. Please update your dependencies.
I have just fixed the other issue.
